### PR TITLE
Closes #90 #91 Update PPCAT and add urine records

### LINKED ISSUE #90: Feature Request:  Add urine records to PC and PP.
### Feature Idea

Urine records should be added to PC and PP to resolve issues in pharmaverseadam  [https://github.com/pharmaverse/pharmaverseadam/issues/57](https://github.com/pharmaverse/pharmaverseadam/issues/57)

### Relevant Input

_No response_

### Relevant Output

_No response_

### Reproducible Example/Pseudo Code

_No response_

### LINKED ISSUE #91: Feature Request:  Update PP.PPCAT to more meaningful values
### Feature Idea

I think PPCAT is usually the analyte name.  Here it is listed as "COMPARTMENTAL", it may be useful to update this value.

May help resolve https://github.com/pharmaverse/pharmaverseadam/issues/57

### Relevant Input

_No response_

### Relevant Output

_No response_

### Reproducible Example/Pseudo Code

_No response_

### PR DESCRIPTION
**Pull request by jeffreyad**
Thank you for your Pull Request! We have developed this task checklist from the [Development Process Guide](https://pharmaverse.github.io/admiraldev/devel/articles/development_process.html) to help with the final steps of the process. Completing the below tasks helps to ensure our reviewers can maximize their time on your code as well as making sure the admiral codebase remains robust and consistent.   

Please check off each taskbox as an acknowledgment that you completed the task or check off that it is not relevant to your Pull Request. This checklist is part of the Github Action workflows and the Pull Request will not be merged into the `devel` branch until you have checked off each task.

- [x] Place Closes #<insert_issue_number> into the beginning of your Pull Request Title (Use Edit button in top-right if you need to update)
- [x] Code is formatted according to the [tidyverse style guide](https://style.tidyverse.org/). Run `styler::style_file()` to style R and Rmd files
- [x] Updated relevant unit tests or have written new unit tests, which should consider realistic data scenarios and edge cases, e.g. empty datasets, errors, boundary cases etc. - See [Unit Test Guide](https://pharmaverse.github.io/admiraldev/devel/articles/unit_test_guidance.html#tests-should-be-robust-to-cover-realistic-data-scenarios)
- [x] If you removed/replaced any function and/or function parameters, did you fully follow the [deprecation guidance](https://pharmaverse.github.io/admiraldev/devel/articles/programming_strategy.html#deprecation)?
- [x] Update to all relevant roxygen headers and examples, including keywords and families. Refer to the [categorization of functions](https://pharmaverse.github.io/admiraldev/devel/articles/programming_strategy.html#categorization-of-functions) to tag appropriate keyword/family.
- [x] Run `devtools::document()` so all `.Rd` files in the `man` folder and the `NAMESPACE` file in the project root are updated appropriately
- [x] Address any updates needed for vignettes and/or templates
- [x] Update `NEWS.md` if the changes pertain to a user-facing function (i.e. it has an `@export` tag) or documentation aimed at users (rather than developers)
- [x] Build admiral site `pkgdown::build_site()` and check that all affected examples are displayed correctly and that all new functions occur on the "[Reference](https://pharmaverse.github.io/admiral/cran-release/reference/index.html)" page. 
- [x] Address or fix all lintr warnings and errors - `lintr::lint_package()`
- [x] Run `R CMD check` locally and address all errors and warnings - `devtools::check()`
- [x] Link the issue in the Development Section on the right hand side.
- [x] Address all merge conflicts and resolve appropriately 
- [x] Pat yourself on the back for a job well done! Much love to your accomplishment!

### CODE REVIEW

File: data-raw/pp.R [hunk 1]
```diff
@@ -144,7 +144,7 @@ pp_tmax$PPORRES <- pp_tmax$TMAX
 PP <- bind_rows(pp_tmax, pp_npts, pp_lambda, pp_Ke, pp_cmax, pp_Clast, pp_AUC, pp_AUC_inf)
 
 # Constant variables
-PP$PPCAT <- "COMPARTMENTAL"
+PP$PPCAT <- unique(pc$PCTEST)
```
Comment (kaz462) [inline data-raw/pp.R:147]: after this update, `PPCAT` contains one value - `XANOMELINE` for all records - is this the intention?

```r
> count(pp, PPCAT)
       PPCAT    n
1 XANOMELINE 2352
```

**Review by kaz462 — COMMENTED** [review]
@jeffreyad Thanks a lot for updating PC/PP! 
Ready to merge it after your confirmation

File: data-raw/pp.R (hunk 1 above)
Comment (jeffreyad) [reply data-raw/pp.R:147]: Still thinking about adding URINE records too

Comment (manciniedoardo) [reply data-raw/pp.R:147]: I think it would be good to add some urine data too if at all possible please

**Review by kaz462 — COMMENTED** [review]
@jeffreyad Thanks for adding the urine records! Just left one question, otherwise it looks good to me.

File: data-raw/pc.R [hunk 2]
```diff
@@ -83,10 +86,28 @@ PC$PCSPEC <- "PLASMA"
...
+PC_Urine <- PC %>%
+  filter(PCTPTNUM %in% c(6.00, 12.00, 24.00, 48.00)) %>%
+  mutate(
+    PCSPEC = "URINE",
+    PCTPT = case_when(
+      PCTPT == "6h Post-dose" ~ "0-6h Post-dose",
+      PCTPT == "12h Post-dose" ~ "6-12h Post-dose",
+      PCTPT == "24h Post-dose" ~ "12-24h Post-dose",
+      PCTPT == "48h Post-dose" ~ "24-48h Post-dose"
+    ),
```
[the remark this answers is not in the document]
Comment (jeffreyad) [reply data-raw/pc.R:106]: Hi @kaz462 Thanks, I will update so that PCTPTNUM is unique.

**Review by manciniedoardo — APPROVED** [review]

**Review by kaz462 — APPROVED** [review]

**Pull request merged**